src/Groups/pcgroup.jl

```diff
@@ -365,3 +365,57 @@ function pc_group(c::GAP_Collector)
   end
 end
 
+"""
+    letters(g::PcGroupElem)
+
+Return the letters of `g` as a list of integers, each entry corresponding to
+a group generator.
+
+# Examples
+```jldoctest
+julia> c = collector(2, Int);
+
+julia> Oscar.set_relative_orders!(c, [2, 3])
+
+julia> Oscar.set_conjugate!(c, 2, 1, [2 => 2])
+
+julia> gg = pc_group(c)
+Pc group of order 6
+
+julia> letters(gg[1]^5*gg[2]^-4)
+3-element Vector{Int64}:
+ 1
+ 2
+ 2
+```
+"""
+function letters(g::PcGroupElem)
+  w = GAPWrap.UnderlyingElement(GapObj(g))
+  return Vector{Int}(GAPWrap.LetterRepAssocWord(w))
+end 
+
+function Oscar.syllables(g::Union{PcGroupElem, SubPcGroupElem})
+  l = GAPWrap.ExtRepOfObj(GapObj(g))
+  @assert iseven(length(l))
+  return Pair{Int, ZZRingElem}[l[i-1] => l[i] for i = 2:2:length(l)]
+end
+
+# Convert syllables in canonical form into exponent vector
+#Thomas
+function exponent_vector(sylls::Vector{Pair{Int64, ZZRingElem}}, n)
```

Comment:
I don't like that this adds a method with basic data types to an exported function that has a higher-level meaning for data structures like polynomials.
```julia
julia> methods(exponent_vector)
# 12 methods for generic function "exponent_vector" from AbstractAlgebra.Generic:
  [1] exponent_vector(::Type{ZZRingElem}, a::Union{FpMPolyRingElem, QQMPolyRingElem, ZZMPolyRingElem, fpMPolyRingElem, fqPolyRepMPolyRingElem, zzModMPolyRingElem}, i::Int64)
     @ Nemo ~/.julia/packages/Nemo/0X3AB/src/flint/mpoly.jl:28
  [2] exponent_vector(a::FqMPolyRingElem, i::Int64)
     @ Nemo ~/.julia/packages/Nemo/0X3AB/src/flint/fq_default_mpoly.jl:486
  [3] exponent_vector(::Type{T}, a::Union{FpMPolyRingElem, QQMPolyRingElem, ZZMPolyRingElem, fpMPolyRingElem, fqPolyRepMPolyRingElem, zzModMPolyRingElem}, i::Int64) where T<:Union{Int64, UInt64}
     @ Nemo ~/.julia/packages/Nemo/0X3AB/src/flint/mpoly.jl:18
  [4] exponent_vector(p::AbstractAlgebra.Generic.UnivPoly, i::Int64)
     @ ~/.julia/packages/AbstractAlgebra/DRky3/src/generic/UnivPoly.jl:59
  [5] exponent_vector(a::MPolyDecRingElem, i::Int64)
     @ Oscar ~/code/julia/Oscar.jl/src/Rings/mpoly-graded.jl:739
  [6] exponent_vector(a::MPolyDecRingElem, i::Int64, ::Type{T}) where T
     @ Oscar ~/code/julia/Oscar.jl/src/Rings/mpoly-graded.jl:740
  [7] exponent_vector(a::AbstractAlgebra.Generic.MPoly{T}, i::Int64, ::Val{:lex}) where T<:RingElement
     @ ~/.julia/packages/AbstractAlgebra/DRky3/src/generic/MPoly.jl:120
  [8] exponent_vector(a::AbstractAlgebra.Generic.MPoly{T}, i::Int64, ::Val{:deglex}) where T<:RingElement
     @ ~/.julia/packages/AbstractAlgebra/DRky3/src/generic/MPoly.jl:130
  [9] exponent_vector(a::AbstractAlgebra.Generic.MPoly{T}, i::Int64, ::Val{:degrevlex}) where T<:RingElement
     @ ~/.julia/packages/AbstractAlgebra/DRky3/src/generic/MPoly.jl:140
 [10] exponent_vector(a::AbstractAlgebra.Generic.MPoly{T}, i::Int64) where T<:RingElement
     @ ~/.julia/packages/AbstractAlgebra/DRky3/src/generic/MPoly.jl:158
 [11] exponent_vector(a::AbstractAlgebra.Generic.MPoly{T}, i::Int64, ::Type{Val{ord}}) where {T<:RingElement, ord}
     @ AbstractAlgebra deprecated.jl:103
 [12] exponent_vector(a::Union{FpMPolyRingElem, QQMPolyRingElem, ZZMPolyRingElem, fpMPolyRingElem, fqPolyRepMPolyRingElem, zzModMPolyRingElem}, i::Int64)
     @ Nemo ~/.julia/packages/Nemo/0X3AB/src/flint/mpoly.jl:38
```
maybe move it to an unexported helper function under a different name? (like e.g. `_exponent_vector`)

Reply:
The idea is to manipulate low level data that describe group elements, in order to create some other group elements. I think this is not intended as a user function, thus it need not be exported.

Reply:
Thanks for the reply @ThomasBreuer. The problem is that the same name as some existing exported function automatically exports this as well. @jamesnohilly can you take care of renaming this function to e.g. `_exponent_vector` or something similar?

Reply:
No problem! I've renamed the function and its usages to `_exponent_vector` as of the latest commit.